src/components/Auth/SignupPage/components/CustomInput/index.tsx

```
@@ -6,9 +6,11 @@ import ErrorIcon from 'assets/svg/Login/error.svg';
 import EyeCloseIcon from 'assets/svg/Login/eye-close.svg';
 import EyeOpenIcon from 'assets/svg/Login/eye-open.svg';
 import WarningIcon from 'assets/svg/Login/warning.svg';
+import MobileWarningIcon from 'assets/svg/mobile-warning-icon.svg';
 import FormatTime from 'components/Auth/SignupPage/hooks/useFormatTime';
 import { useFormContext } from 'react-hook-form';
```

ESLint

[error] `react-hook-form` import should occur before import of `assets/svg/Login/close.svg`

(import/order)